core/services/ocr2/plugins/median/config/config.go

```diff
@@ -14,9 +14,10 @@ import (
 
 // The PluginConfig struct contains the custom arguments needed for the Median plugin.
 type PluginConfig struct {
-	JuelsPerFeeCoinPipeline      string          `json:"juelsPerFeeCoinSource"`
-	JuelsPerFeeCoinCacheDuration models.Interval `json:"juelsPerFeeCoinCacheDuration"`
-	JuelsPerFeeCoinCacheDisabled bool            `json:"juelsPerFeeCoinCacheDisabled"`
+	JuelsPerFeeCoinPipeline            string          `json:"juelsPerFeeCoinSource"`
+	JuelsPerFeeCoinCacheDuration       models.Interval `json:"juelsPerFeeCoinCacheDuration"`
+	JuelsPerFeeCoinCacheStalenessAlert models.Interval `json:"juelsPerFeeCoinCacheStalenessAlert"`
+	JuelsPerFeeCoinCacheDisabled       bool            `json:"juelsPerFeeCoinCacheDisabled"`
```

Comment:
Perhaps it is too late, but would a nested struct make things easier?
```suggestion
	JuelsPerFeeCointCache JuelsPerFeeCointCache `json:"juelsPerFeeCoinCache"`
}
type JuelsPerFeeCointCache struct {
	Duration       models.Interval `json:"duration"`
	StalenessAlert models.Interval `json:"stalenessAlert"`
	Disabled       bool            `json:"disabled"`
```